go.mod

```
@@ -114,4 +114,4 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
-go 1.23
+go 1.24.1
```

```suggestion
go 1.23
```

This doesn't have to be modified yet - it's just the minimum Go version required to build the module (and in any case it can't contain a patch release number).

---

Actually it appears it _can_ contain a patch release number (TIL)...